src/main/java/uk/gov/hmcts/dm/controller/StoredDocumentController.java

```diff
@@ -99,11 +106,23 @@ private void init() throws Exception {
 
     @DeleteMapping(value = "{id}")
     @ApiOperation("(Soft) Deletes a Stored Document.")
-    @ApiResponses(value={
-        @ApiResponse(code=405, message = "Method not implemented at the moment")
-    })
+
     public ResponseEntity<Object> delete(@PathVariable UUID id) {
-        return ResponseEntity.status(405).build();
+        if (deleteEnabled) {
+            auditedStoredDocumentOperationsService.deleteStoredDocument(id);
+            return ResponseEntity.status(HttpStatus.GONE).build();
```

Comment:
`GONE` is an error status, I assume this should be a success response.

Reply:
Hey. Yep, I've updated the response status to be 204 for success.